src/extension/extension.ts

```diff
@@ -354,6 +357,34 @@ export class RunmeExtension {
     }
 
     await bootFile(context)
+
+    if (getBetaFlag<boolean>('shellWarning', false)) {
+      const showUnsupportedShellMessage = async () => {
+        const showMore = 'Show more'
+
+        const answer = await window.showErrorMessage('Unsupported shell', showMore)
+        if (answer === showMore) {
+          const url = getDocsUrl('/r/extension/supported-shells')
+          env.openExternal(Uri.parse(url))
+        }
+      }
+
+      kernel
+        .runProgram('echo $SHELL')
+        .then(({ output }) => {
+          const supportedShells = ['bash']
```

Comment:
@pastuxso no more `zsh`?

Reply:
Thank you for catching that. I thought I had added it back, but it seems I missed it. I’ll correct it right away. I appreciate your attention to detail.